logstash-input-kafka.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |s|
 
   s.requirements << "jar 'org.apache.kafka:kafka-clients', '0.11.0.0'"
   s.requirements << "jar 'org.apache.logging.log4j:log4j-slf4j-impl', '2.8.2'"
+  s.requirements << "jar 'io.confluent:kafka-avro-serializer', '3.3.0'"
```

Comment:
I believe this can only be found outside of maven central in confluent's maven repository, does this work as it is for you?

Reply:
I can't seem to make this PR work at all, even after hacking the repositories to include confluent's, I'm still getting:

```
[2018-06-06T21:43:28,734][ERROR][logstash.inputs.kafka    ] Unable to create Kafka consumer from given configuration {:kafka_error_message=>#<NameError: cannot link Java class io.confluent.kafka.serializers.KafkaAvroDeserializerConfig (java.lang.NoClassDefFoundError: io/confluent/common/config/AbstractConfig)>, :cause=>nil}
```

Reply:
@jsvd Apparently the jar was already in my local maven repository and in our nexus so the build always worked locally and on our build servers.  I tried to run a build in a fresh jruby docker container and it failed because it isn't in maven central.  Sorry about that.

Reply:
are you using the standard kafka oss distribution or confluent's?

Reply:
@jsvd We are using the standard kafka oss distribution.  I haven't run into that error, I'm guessing that it is because whatever you did to hack the repositories prevented maven transitive dependencies from getting included.

Reply:
@jsvd Here is what the lib directory looks like after a build for me, the lib/io/confluent/common-config/3.3.0/common-config-3.3.0.jar should have the class that was missing from your build.

```
root@9c6bee2f647c:/mnt# find lib
lib
lib/logstash-input-kafka_jars.rb
lib/jline
lib/jline/jline
lib/jline/jline/0.9.94
lib/jline/jline/0.9.94/jline-0.9.94.jar
lib/com
lib/com/fasterxml
lib/com/fasterxml/jackson
lib/com/fasterxml/jackson/core
lib/com/fasterxml/jackson/core/jackson-databind
lib/com/fasterxml/jackson/core/jackson-databind/2.8.4
lib/com/fasterxml/jackson/core/jackson-databind/2.8.4/jackson-databind-2.8.4.jar
lib/com/fasterxml/jackson/core/jackson-core
lib/com/fasterxml/jackson/core/jackson-core/2.8.4
lib/com/fasterxml/jackson/core/jackson-core/2.8.4/jackson-core-2.8.4.jar
lib/com/fasterxml/jackson/core/jackson-annotations
lib/com/fasterxml/jackson/core/jackson-annotations/2.8.0
lib/com/fasterxml/jackson/core/jackson-annotations/2.8.0/jackson-annotations-2.8.0.jar
lib/com/thoughtworks
lib/com/thoughtworks/paranamer
lib/com/thoughtworks/paranamer/paranamer
lib/com/thoughtworks/paranamer/paranamer/2.7
lib/com/thoughtworks/paranamer/paranamer/2.7/paranamer-2.7.jar
lib/com/101tec
lib/com/101tec/zkclient
lib/com/101tec/zkclient/0.10
lib/com/101tec/zkclient/0.10/zkclient-0.10.jar
lib/log4j
lib/log4j/log4j
lib/log4j/log4j/1.2.17
lib/log4j/log4j/1.2.17/log4j-1.2.17.jar
lib/org
lib/org/xerial
lib/org/xerial/snappy
lib/org/xerial/snappy/snappy-java
lib/org/xerial/snappy/snappy-java/1.1.2.6
lib/org/xerial/snappy/snappy-java/1.1.2.6/snappy-java-1.1.2.6.jar
lib/org/apache
lib/org/apache/zookeeper
lib/org/apache/zookeeper/zookeeper
lib/org/apache/zookeeper/zookeeper/3.4.8
lib/org/apache/zookeeper/zookeeper/3.4.8/zookeeper-3.4.8.jar
lib/org/apache/kafka
lib/org/apache/kafka/kafka-clients
lib/org/apache/kafka/kafka-clients/0.11.0.0
lib/org/apache/kafka/kafka-clients/0.11.0.0/kafka-clients-0.11.0.0.jar
lib/org/apache/avro
lib/org/apache/avro/avro
lib/org/apache/avro/avro/1.8.2
lib/org/apache/avro/avro/1.8.2/avro-1.8.2.jar
lib/org/apache/commons
lib/org/apache/commons/commons-compress
lib/org/apache/commons/commons-compress/1.8.1
lib/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.jar
lib/org/apache/logging
lib/org/apache/logging/log4j
lib/org/apache/logging/log4j/log4j-api
lib/org/apache/logging/log4j/log4j-api/2.8.2
lib/org/apache/logging/log4j/log4j-api/2.8.2/log4j-api-2.8.2.jar
lib/org/apache/logging/log4j/log4j-slf4j-impl
lib/org/apache/logging/log4j/log4j-slf4j-impl/2.8.2
lib/org/apache/logging/log4j/log4j-slf4j-impl/2.8.2/log4j-slf4j-impl-2.8.2.jar
lib/org/codehaus
lib/org/codehaus/jackson
lib/org/codehaus/jackson/jackson-core-asl
lib/org/codehaus/jackson/jackson-core-asl/1.9.13
lib/org/codehaus/jackson/jackson-core-asl/1.9.13/jackson-core-asl-1.9.13.jar
lib/org/codehaus/jackson/jackson-mapper-asl
lib/org/codehaus/jackson/jackson-mapper-asl/1.9.13
lib/org/codehaus/jackson/jackson-mapper-asl/1.9.13/jackson-mapper-asl-1.9.13.jar
lib/org/slf4j
lib/org/slf4j/slf4j-log4j12
lib/org/slf4j/slf4j-log4j12/1.7.21
lib/org/slf4j/slf4j-log4j12/1.7.21/slf4j-log4j12-1.7.21.jar
lib/org/slf4j/slf4j-api
lib/org/slf4j/slf4j-api/1.7.25
lib/org/slf4j/slf4j-api/1.7.25/slf4j-api-1.7.25.jar
lib/org/tukaani
lib/org/tukaani/xz
lib/org/tukaani/xz/1.5
lib/org/tukaani/xz/1.5/xz-1.5.jar
lib/io
lib/io/netty
lib/io/netty/netty
lib/io/netty/netty/3.7.0.Final
lib/io/netty/netty/3.7.0.Final/netty-3.7.0.Final.jar
lib/io/confluent
lib/io/confluent/common-config
lib/io/confluent/common-config/3.3.0
lib/io/confluent/common-config/3.3.0/common-config-3.3.0.jar
lib/io/confluent/kafka-avro-serializer
lib/io/confluent/kafka-avro-serializer/3.3.0
lib/io/confluent/kafka-avro-serializer/3.3.0/kafka-avro-serializer-3.3.0.jar
lib/io/confluent/kafka-schema-registry-client
lib/io/confluent/kafka-schema-registry-client/3.3.0
lib/io/confluent/kafka-schema-registry-client/3.3.0/kafka-schema-registry-client-3.3.0.jar
lib/io/confluent/common-utils
lib/io/confluent/common-utils/3.3.0
lib/io/confluent/common-utils/3.3.0/common-utils-3.3.0.jar
lib/net
lib/net/jpountz
lib/net/jpountz/lz4
lib/net/jpountz/lz4/lz4
lib/net/jpountz/lz4/lz4/1.3.0
lib/net/jpountz/lz4/lz4/1.3.0/lz4-1.3.0.jar
lib/logstash
lib/logstash/inputs
lib/logstash/inputs/kafka.rb
```